backend/src/main/java/de/tostsoft/solarmonitoring/service/SolarSystemService.java

```diff
@@ -134,8 +134,12 @@ public void deleteSystem(String token) throws Exception {
       throw new Exception("System not exist");
 
     }
-    if (user.getRelationOwns().contains(solarSystem)) {
-      solarSystemRepository.deleteByToken(token);
+    for (int i = 0;user.getRelationOwns().size()>i;i++){
```

Comment:
keine forschleifen mit zahlen als itteratoren wenn diese nicht benötig werden in der schleife benutze diese java syntax: for (Object object : objects) {
    object.toSomething()
}